terraform/README.rst

```diff
@@ -0,0 +1,56 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Terraform: Airflow infrastructure as code
+========================================
+
+In this folder, you will find the Terraform code to deploy the Airflow infrastructure on AWS.
+
+The `eks` folder contains the code to deploy the Airflow EKS cluster and all the required resources.
+
+Requirements
+------------
+
+In order to deploy the infrastructure, you need to have the following tools installed:
+
+- `tfenv <https://github.com/tfutils/tfenv>`_ to manage Terraform versions
+- Configure your AWS and EKS credentials :ref:`doc <aws_configuration>`
+
+
+Configure Terraform
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+   cd eks
+   tfenv install 1.5.3
```

Comment:
It's not a deal breaker, just a small good to have. We can include a `.terraform-version` file to manage the version to not check from `README`. We can then call `tfenv install` to install the correct version without specifying. This can be done in a later stage as well after this one is merged.

Reply:
I pinned the version, I will update the readme as we don't need to install it manually anymore